README.md

```diff
@@ -66,24 +147,24 @@ state, and are less stable than the `tracing` and `tracing-core` crates.
 
 The crates included as part of Tracing are:
 
-* [`tracing-futures`]: Utilities for instrumenting `futures`.
+- [`tracing-futures`]: Utilities for instrumenting `futures`.
```

Comment:
The change in markdown here seems unrelated to the rest of the change...was this suggested by `markdownlint` or something?

Reply:
ah, my bad, I think that's my lint doing.. will undo the changes